content/blog/2022-06-29-june-community-gems.md

```diff
@@ -0,0 +1,200 @@
+download anything to your local cache so it's another way to transfer data
+between remotes.
+
+## [If an image is present in different directories in different projects, will the shared cache store them both as one hash or will their different paths mean the same image appears twice in the cache?](https://discord.com/channels/485586884165107732/563406153334128681/984408209387298837)
+
+Great question about the cache from @paulwrightkcl!
+
+DVC will index the whole directory, but there will only be one hash per file. So
+the same image will only appear once in the cache. What _will_ be duplicated in
+the cache is the `.dir` hash that DVC uses internally as the directory tree
+representation.
+
+In summary, the image file is only stored in the shared cache once unless it's
+modified in one of the directories.
+
+## [Is it possible to limit which columns show for experiments in the metrics table?](https://discord.com/channels/485586884165107732/563406153334128681/985448515402616842)
+
+Nice question from @DylanTF!
+
+You can use `dvc exp --drop` (or `--keep`) to decide what to hide (or show). For
+example, if you have a table like this:
+
+```dvctable
+ ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+  neutral:**Experiment**   neutral:**Created**        metric:**avg_prec**   metric:**roc_auc**   param:**train.seed**   param:**train.n_est**   param:**train.min_split**   dep:**./clf**   dep:**./data**    dep:**./data/train.pkl**   dep:**./src/train.py**   dep:**src/evaluate.py**
+ ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+  workspace    -                     -         -   20210428     300           75                -       a9bb63e   aded63c            bdc3fe9          b0ef2a1
+  mlem-serve   Jun 16, 2022    0.76681   0.38867   20210428     300           75                -       a9bb63e   aded63c            bdc3fe9          b0ef2a1
+ ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+```
+
+You could clean it up with a command like this:
+
+```dvc
+$ dvc exp show --drop 'Created|train.seed|./clf|./data/*|./src/train.py|src/evaluate.py'
+```
+
+Then get a table like this:
+
+```dvctable
+ ─────────────────────────────────────────────────────────────────
+  neutral:**Experiment**   metric:**avg_prec**   metric:**roc_auc**   param:**train.n_est**   param:**train.min_split**
+ ─────────────────────────────────────────────────────────────────
+  workspace           -         -   300           75
+  mlem-serve    0.76681   0.38867   300           75
+ ─────────────────────────────────────────────────────────────────
+```
+
+If you're interested in more advanced visualizations, you should try out
```

Comment:
This seems like a prime candidate to plug the VSCode extension as well! Do the same but with a mouse click. 😄